### ISSUE TITLE
Fix or remove broken designs

### ISSUE THREAD

**Issue by selfthinker** [selfthinker/CSS1K#1]
Some designs are broken. Contact authors and ask them if they are interested in fixing them. If not, either fix myself or remove them.

**Comment by selfthinker** [comment]
- [x] notepad: background lines
- [x] gardenof3den: page background and leaves background
- [x] facetime: the copy inside the glasses doesn't unblur when hovering over it
- [x] blue_square: main copy doesn't fit into screen (on smaller but still desktop screens)
- [x] tangerine: cannot select a design
- [x] solarized: the first heading in the article starts floating on the right of the design list
- [x] html5colors: broken background
- [x] b4ya11y: the design list overlaps the main content (on smaller but still desktop screens)
- [x] expandable: main copy is unreadable
- [x] bookshelf: no horizontal scrolling
- [x] openwater: gradient making the deep water effect and waves on the 'surface' have gone

Due to added copy (my fault):
- [x] moonlight: background to intro not fitting
- [x] cloudy: the cloud is overlapping "Select a Design"
- [x] tangerine: beginning of intro is hidden behind the main header (on smaller but still desktop screens)
- [x] triangulate: the title is overlapping the intro

**Comment by SelenIT** [comment]
The most part of broken designs are because of old Prefix-free version that uses old gradient syntax (without 'to' keyword). Probably this [update](http://lea.verou.me/2012/07/important-prefix-free-update/) should fix the issue, although prefixes for gradients looks rather unnecessary at all in 2015.

My design (bookshelf) also appeared to be broken in Chrome (no horizontal scrolling), I'll fix it soon.

And thank you very much for reviving this project!

**Comment by selfthinker** [comment]
I had already updated PrefixFree. (It's StyleFix 1.0.3 & PrefixFree 1.0.7 now.)

I'm pretty sure the above issues will be relatively easy to fix, although I haven't looked at them yet. The only difficult thing is to work with minified CSS. (I know, there are tools to un-minify, but it still seems to be an unnecessary step.)

**Comment by SelenIT** [comment]
At some point in 2011, all designs were stored in the project in 2 versions, both minified and unminified.   Some unminified styles are available in https://github.com/SelenIT/CSS1K/tree/master/styles, probably other style authors have their own (including most recent ones). Hope this helps.

**Comment by SelenIT** [comment]
I revisited the issue with gradients and found out that Prefix-free doesn't convert unprefixed old syntax (without 'to' for linear gradients/'at' for radial ones) to modern syntax. Almost all pre-2012 designs suffer from this limitation (in addition to your list: Open Water — gradient making the deep water effect and waves on the 'surface' have gone). I believe this can be easy fixed, e.g. with [postcss-gradientfixer](https://github.com/hallvors/postcss-gradientfixer) (probably there are other tools for this that can make it even easier).

**Comment by selfthinker** [comment]
I just made a tick list of the list of issues and added the other two you found (bookshelf and openwater, with bookshelf being ticked already because you fixed it in #4).

I saved the old original files into a new folder, but it's currently only in a local branch. I'm not sure if it's helpful when the minification is not automatic.

I will go about fixing a few of the others now.

**Comment by selfthinker** [comment]
I managed to fix nearly everything.
What I couldn't fix:
- tangerine: The bug that you cannot select any other design must have been from there from the very beginning. I am considering just removing it for that reason. The basic functionality of the page (which is basically just copy and links) needs to be accessible.
- northern-lights: That it's difficult to access the bottom part of the design list is annoying but probably not a deal breaker. I will just leave it as it is for now.
- facetime: I managed to fix the issue but couldn't yet find a way to keep it under 1024 bytes. I wonder why it was ever working (and I remember it working)?
- b4ya11y: I only fixed it so that it works on 1260px wide screens. Anything below still makes the copy overlap. I wonder if I should remove it.

I think there should be clearer instructions on what the requirements are.
I'd propose:
- At least working in a 1024x768 screen. (Ideally I would want everything to work on mobile devices but that might be unreasonable for such a small CSS file.)
- All links must be mouse-accessible. Should all links be also keyboard-accessible? Lots of current effects would suffer then.
- Should all copy be keyboard-accessible? Again, especially the more experimental effects would not be possible.
- Everything needs to work in modern browsers. I think just latest Chrome, Firefox and Edge should be enough.
